src/messages-codes.h.jinja

```diff
@@ -6,6 +6,14 @@
 
 #include <stdint.h>
 
+/**
+ * Special case when no message identifier is defined.
+ *
+ * @added 1.6.0
```

Comment:
This is not public API so the `@added` is not needed

Reply:
Right! I also removed it from `enum xkb_message_code`.